cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -441,3 +299,283 @@ def test_conversion_host_tags(appliance, v2v_providers):
+    for vm in vms:
+        popup_text = request_details_list.read_additional_info_popup(vm)
+        # open__additional_info_popup function also closes opened popup in our case
+        request_details_list.open_additional_info_popup(vm)
+        if popup_text['Conversion Host'] in hosts_dict:
+            hosts_dict[popup_text['Conversion Host']].append(vm)
+    for host in hosts_dict:
+        logger.info("Host: {} is migrating VMs: {}".format(host, hosts_dict[host]))
+        assert len(hosts_dict[host]) > 0, ("Conversion Host: {} not being utilized for migration!"
+            .format(host))
+
+    wait_for(func=view.get_plan_status, message="migration plan is in progress, be patient please",
+     delay=5, num_sec=14400)
+
+    for vm in vms:
+        soft_assert(request_details_list.is_successful(vm) and
+         not request_details_list.is_errored(vm))
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+ indirect=True)
+def test_single_vm_scheduled_migration(request, appliance, v2v_providers, host_creds,
+                                    conversion_tags, soft_assert,
+                                    form_data_vm_obj_single_datastore):
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+
+    mapping = infrastructure_mapping_collection.create(
+        form_data_vm_obj_single_datastore[0])
+
+    @request.addfinalizer
+    def _cleanup():
+        infrastructure_mapping_collection.delete(mapping)
+
+    migration_plan_collection = appliance.collections.v2v_plans
+
+    migration_plan = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()), description="desc_{}"
+        .format(fauxfactory.gen_alphanumeric()), infra_map=mapping.name,
+        vm_list=form_data_vm_obj_single_datastore[1], start_migration=False)
+    view = appliance.browser.create_view(navigator.get_class(migration_plan_collection, 'All').VIEW)
+
+    # Test scheduled Migration
+    view.migration_plans_not_started_list.schedule_migration(migration_plan.name, after_mins=2)
+    soft_assert('Migration scheduled' in view.migration_plans_not_started_list.
+        get_clock(migration_plan.name))
+    view.migration_plans_not_started_list.schedule_migration(migration_plan.name)
+    soft_assert(view.migration_plans_not_started_list.get_clock(migration_plan.name) is None)
+    view.migration_plans_not_started_list.schedule_migration(migration_plan.name, after_mins=1)
+    view.migr_dropdown.item_select('In Progress Plans')
+    wait_for(func=view.progress_card.is_plan_started, func_args=[migration_plan.name],
+        message="migration plan is starting, be patient please", delay=5, num_sec=150,
+        handle_exception=True)
+
+    # wait until plan is in progress
+    wait_for(func=view.get_plan_status, func_args=[migration_plan.name],
+        message="migration plan is in progress, be patient please",
+        delay=5, num_sec=3600)
```

Comment:
heck of a wait time, let's cut these down to 1800 please

Reply:
I will reduce time out for all those tests that are migrating small VM(only 1 VM). For some tests, there are more than 1 VMs being migrated and those are big in size. 30 mins may fall short.